### ISSUE TITLE
NPC: Enraged Stanley is giving too little XP

### ISSUE THREAD

**Issue by Simp-N** [TeamEverlook/Everlook-BugTracker#354]
tl;dr: Enraged Stanley is giving as much XP as any mob of his level, when he should be giving ~7 times more.

At the end of the quest [Elixir of Pain](https://www.wowhead.com/classic/quest=502/elixir-of-pain) you have to feed the elixir to the dog [Stanley](https://www.wowhead.com/classic/npc=2274/stanley), which is done by simply turning the quest in to him. When this is done, Stanley turns into [Enraged Stanley](https://www.wowhead.com/classic/npc=2275/enraged-stanley), who is hostile and can be killed. When killed Enraged Stanley should reward a lot more XP than usual mobs of his level, almost as much as a quest gives. However, on Everlook he currently just gives as much XP as any other mob of his level.

Proof that he should give much more XP can be found on numerous comments at WoWhead from vanilla. Comments from Wowhead on the [quest](https://www.wowhead.com/classic/quest=502/elixir-of-pain#comments):
> I got 682 exp for killing him (in addition to 1150 for the quest) I was level 30. If you can definataly kill him.

>Give it to the dog, and you get a little bit of xp, and then it turns into a worg looking mob.  Kill that and you get like 2-3k xp.  Nice set of quests.

>Soloed with level 26 rogue, stay there back up and kill the dog, pat goes left wait for that, then feed dog back up kill dog I got 1002 xp for killing a mob same xp as I, and wasn't even rested

> Don't run away after you gave the elixir to standley , the dog , if you kill him , u get around 900-1100xp just for him + the quest exp. Means around 2500xp.

> Just kill the dog, you get a alot of XP.

> When you turn in the quest he turns into a level 24 monster, make sure to kill him as he gives 700 exp or so when killed.

> killing the wolflike mob that stanley turns into after you give him the potion also gives you ~1000 xp

> Doing this quest while rested is a very good idea. I got 2000+ xp from killing the dog at level 25, 1000 or so of that from rested XP. Not to mention the 1500 XP from doing the quest itself.

Comments from Wowhead on the [NPC](https://www.wowhead.com/classic/npc=2275/enraged-stanley#comments):
> Quest chain "Elixir of Pain" from the apothecary in Tarren Mill will have you poison Stanley, which makes him 'Enraged Stanley'. At this point you can kill him for excessive xp, or tame him.

> I got 1082xp for killing this dog. I also got 1450xp for completing the quest. How in the hell?

He also gives a lot of xp in Classic, as shown in this video: https://youtu.be/Jz84lLlG2AU?si=iJGl9col5Rtdj32y&t=848

Based on a few of these comments and on the Classic video I tried to figure out how much XP he *should* give. We know Enraged Stanley is lvl 24. Together with the players level we can figure out how much xp a regular lvl 24 mob would give, and compare that to what they actually got from Stanley to get a coefficient. I am not sure if his XP formula is simply a multiplier but it's better than nothing. At any rate, taken from some of the comments above: 
lvl 30 player got 682 xp. A lvl 30 would normally get 98 xp from a lvl 24 mob -> 682 / 98 = **6,96**
lvl 26 player got 1002 -> 1002 / 143 = **7,00**

These two examples are the only ones I could find where the player's level is known, and they state an exact XP value. Lasty, here's a cropped screenshot showing he currently only gives regular XP on Everlook: 
![bild](https://github.com/TeamEverlook/Everlook-BugTracker/assets/75734743/3819807e-f494-45d4-b761-11be4ebb74bd)

Hope I didn't bore you with all this but killing Enraged Stanley and not getting that juicy XP is kinda lackluster. Thanks for reading.

**Comment by jensbond420** [comment]
Yep. I can confirm this, added to bug list. Thank you
